unpackaged/pre/case_behavior_record_types/package.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Package xmlns="http://soap.sforce.com/2006/04/metadata">
+  	<types>
```

Comment:
Minor, but it seems like the formatting got messed up a bit here.

Reply:
Should be good now!